operations/service_deploy_smoke_utils.go

```diff
@@ -221,17 +187,15 @@ OUTER:
 			}
 			if task.Status != evergreen.TaskSucceeded {
 				grip.Infof("found task is status %s", task.Status)
-				task = apimodels.APITask{
-					Status: task.Status,
-				}
+				taskStatus = task.Status
 				continue OUTER
 			}
-			err = checkLog(task, client, agent.HostMode, username, key)
+			err = checkLog(task, client, mode, username, key)
 			if err != nil {
 				return err
 			}
 		}
-		grip.Info("Successfully checked tasks")
+		grip.Infof("Successfully checked %s tasks", string(mode))
```

Comment:
Would it help to include the number of tasks that were checked here for debugging?

Reply:
Sure, added `len(tasks)` to the formatted print statement